src/components/Report.js

```diff
@@ -16,7 +16,8 @@ const storyColumns = [
   ['Title', 'title'],
   ['Estimate', 'estimate'],
   ['Opened At', v => shortDay(v.openedAt)],
-  ['Closed At', v => shortDay(v.closedAt)],
+  ['Closed At', v => (v.closedAt ? shortDay(v.closedAt) : '')],
```

Comment:
There's a commit in @kalongthewires's PR #25 (fb62324a9fa61c2e97978147078ef35e8906d431) that addresses this issue more comprehensively; there were some issues with date comparison since we weren't clamping to date rather than datetime.

Maybe we should leave this out of your PR and get @kalongthewires's merged to avoid duplication / merge issues?